cmd/syncthing/usage_report.go

```diff
@@ -182,8 +182,8 @@ func reportData(cfg *config.Wrapper, m *model.Model) map[string]interface{} {
 
 	defaultAnnounceServersDNS, defaultAnnounceServersIP, otherAnnounceServers := 0, 0, 0
 	for _, addr := range cfg.Options().GlobalAnnServers {
-		if addr == "default" {
-			defaultAnnounceServersDNS++
+		if addr == "default" || addr == "default-v4" || addr == "default-v6" {
+			defaultAnnounceServersDNS = 1
```

Comment:
changed :) was also not so sure about what is the best to do here, worst case is 3 if someone sets `default,default-v4,default-v6`